internal/runner/runner.go

```diff
@@ -58,7 +70,7 @@ type Runner struct {
 // namespace configured to the provided values.
 func NewRunner(getter genericclioptions.RESTClientGetter, storageNamespace string, logger logr.Logger) (*Runner, error) {
 	runner := &Runner{
-		logBuffer: NewLogBuffer(NewDebugLog(logger).Log, 0),
+		logBuffer: NewLogBuffer(NewDebugLog(logger).Log, 100),
```

Comment:
I somehow missed this during review, and this change is not related to any of the CRD things. Can you explain why this was changed? As it changes the default of including the last 5 log lines (after de-duplication) in a failure condition and/or event to 100, which makes any failed event we produce now quite verbose.

Reply:
I increased it for debugging during feature development and eventually forgot to decrease it again. 😳

Reply:
Thanks, that means my restore PR is justified and I can continue with the release :-)